packages/algjulia-interop/make_sysimage.jl

```diff
@@ -8,16 +8,17 @@ using PackageCompiler
 Pkg.activate(@__DIR__)
 
 @info "Creating the sysimage. This may take a while..."
-sysimg="AlgebraicJuliaService.so"
-create_sysimage(["AlgebraicJuliaService"], sysimage_path=sysimg,
+sysimg="CatColabInterop.so"
+create_sysimage(["CatColabInterop"], sysimage_path=sysimg,
                 precompile_execution_file="sysimage_precompile.jl")
 sysimg_path=joinpath(@__DIR__, sysimg);
 
 @info "Adding $sysimg_path to IJulia kernel"
 Pkg.activate(); Pkg.add("IJulia")
 using IJulia
+Pkg.build("IJulia")
 
-installkernel("Julia AJaaS", "--project=@.", "--sysimage=$sysimg_path")
+installkernel("Julia CCL Interop", "--project=@.", "--sysimage=$sysimg_path")
```

Comment:
Minor point, but I'd prefer the clearer if slightly more verbose: "Julia CatColab Interop"